src/components/ConcurrencyLimitsV2Table.vue

```diff
@@ -6,10 +6,15 @@
         {{ row.name }}
       </p-link>
     </template>
-    <template #active-task-runs="{ row }">
+
+    <template #active-slots="{ row }">
       {{ row.activeSlots }}
     </template>
 
+    <template #active="{ row }">
+      <ConcurrencyLimitV2Toggle :limit="row" @update="concurrencyLimitSubscription.refresh()" />
```

Comment:
It looks like the toggle is automatically refreshing this subscription. Is this needed? I think this is going to cause the subscription to be refreshed twice.